services/monitoring/grafana/scripts/export.py

```diff
@@ -106,15 +106,25 @@ def main(foldername: str = ""):
         with open(directory + "/alerts/" + alert["name"] + ".json", "w") as outfile:
             print("Export Alert " + alert["name"])
             # Remove UID if present
-            alert["rules"][0]["grafana_alert"].pop("uid", None)
-            # Remove orgId
-            alert["rules"][0]["grafana_alert"].pop("orgId", None)
-            # Remove id
-            alert["rules"][0]["grafana_alert"].pop("id", None)
-            # Remove id
-            alert["rules"][0]["grafana_alert"].pop("namespace_id", None)
-            # Remove id
-            alert["rules"][0]["grafana_alert"].pop("namespace_uid", None)
+            for ruleIter in range(len(alert["rules"])):
```

Comment:
Some TIPS:

1. should start using same coding conventions as other repos (it is far easier to read when we have all the same style!)
2. this loop can simply be done by
```python
for rule in alert['rules']:
    rule["grafana_alert"].pop("uid", None)
   ...

```

Reply:
changed this thx